R/faasr_client_api_aws-lambda.R

```diff
@@ -473,13 +639,46 @@ faasr_set_workflow_timer_ld <- function(faasr, cred, target, cron, unset=FALSE){
     write(targets_json, file = "faasr_lambda_event_targets.json")
     
     # set event target with the faasr_lambda_event_targets.json
-    set_lambda_timer_command <- "aws events put-targets --rule FaaSr-scheduled-cron-rule --targets file://faasr_lambda_event_targets.json"
-    set_lambda_timer_result <- system(set_lambda_timer_command, intern = TRUE)
+
+    set_lambda_timer_result <- eventbridge_instance$put_targets(Rule = event_rule_name, Targets = targets)
+    
     
     file.remove("faasr_lambda_event_targets.json")
   }
   
 }
 
+faasr_workflow_invoke_lambda <- function(faasr, cred, faas_name, actionname){
+    aws_region <- faasr$ComputeServers[[faas_name]]$Region
+    aws_instance <- paws::lambda(
+      config=list(
+      credentials=list(
+        creds=list(
+          access_key_id=cred$lambda_a_ACCESS_KEY,
+          secret_access_key=cred$lambda_a_SECRET_KEY,
+          session_token=""
+        )
+      ),
+      region=aws_region
+    )
+    )
+    # json file with credentials will be created and after invocation, it will be removed.
+    faasr_w_cred <- faasr_replace_values(faasr, cred)
+    faasr_json <- jsonlite::toJSON(faasr_w_cred, auto_unbox=TRUE)
+    rd_nb <- sample(100000, size=1)
+    writeLines(faasr_json, paste0("payload_ld_",rd_nb,".json"))
```

Comment:
We don't need to create the "payload.json" file anymore.
line no.666, 668, and 669 will be unnecessary
line no.682 as well.